marketplace/wpp_templates/models.py

```diff
@@ -0,0 +1,116 @@
+import uuid
+
+from django.db import models
+
+from rest_framework.exceptions import ValidationError
+
+
+class TemplateMessage(models.Model):
```

Comment:
In the planning, the name of the model was `Template`, what is the reason for this change?

Reply:
I think the name Template might be confused